soc/snps/nsim/arc_classic/em/CMakeLists.txt

```diff
@@ -11,18 +11,16 @@ if(COMPILER STREQUAL gcc)
 else()
   # MWDT compiler options
   zephyr_compile_options_ifdef(CONFIG_SOC_NSIM_EM -arcv2em -core3 -Xdiv_rem=radix2
-			       -Xmpy_option=mpyd -Xbitscan -Xswap -Xbarrel_shifter
```

Comment:
It is a bit confusing to have the ARC classic fixes in this ARC-V commit ...

Reply:
I agree